releng/com.zeligsoft.dds4ccm.update.axcioma/rpm_build/zeligsoftCX_axcioma.spec

```diff
@@ -144,6 +147,13 @@ timestamp=$(echo /opt/cx-axcioma/*.zip | sed -rn 's/.*\.v([0-9]*)\.zip/\1/p')
    -repository \
    jar:file:/opt/cx-axcioma/dds4ccm_axcioma_%{ver}.v${timestamp}.zip\!/ \
    -installIU %{feature_groups}
+# Install Papyrus patch
```

Comment:
Sorry, just catching this @elder4p. We'll need an equivalent `-uninstallIU` call for this in the `%postun` section below to uninstall this patch on uninstall.